.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,52 @@
+# Implementing a new supported platform
+
+## At Development
+Create PR's addressing the following:
+- [ ] [coreos-assembler](https://github.com/coreos/coreos-assembler)
+    - [Example PR](https://github.com/coreos/coreos-assembler/pull/2489)
+    - [ ] `Implement required functionality to support new platform`
+- [ ] [fedora-web](https://pagure.io/fedora-web/websites)
+    - [Example PR](https://pagure.io/fedora-web/websites/pull-request/221#request_diff)
+    - [ ] `Add platform to the sites/static/js/coreos-download.js`
+- [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
```

Comment:
```suggestion
- [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
    - [Example PR](https://github.com/coreos/stream-metadata-go/pull/45/)
    - [ ] `Add platform to the Media struct in release/release.go`
    - [ ] `Add supporting code for new platform to _toStreamArch_ func found in the release/translate.go` 
    - [ ] (Cloud only) `Cloud Images need to have a Images field`
- [ ] (Cloud Only) [stream-metadata-rust](https://github.com/coreos/stream-metadata-rust/)
    - [Example PR](ERM)
```

So I was thinking something like this. However, I am not finding a good example of a pr for the stream-metadata-rust adding a new platform. Or I am and just not recognizing it. 

Do you have any recommended Pr's for this?